src/Microsoft.ML.Transforms/CountFeatureSelection.cs

```diff
@@ -13,13 +13,13 @@
 using System.Linq;
 using System.Reflection;
 
-[assembly: LoadableClass(CountFeatureSelectionTransform.Summary, typeof(IDataTransform), typeof(CountFeatureSelectionTransform), typeof(CountFeatureSelectionTransform.Arguments), typeof(SignatureDataTransform),
-    CountFeatureSelectionTransform.UserName, "CountFeatureSelectionTransform", "CountFeatureSelection")]
+[assembly: LoadableClass(CountFeatureSelectionTransformer.Summary, typeof(IDataTransform), typeof(CountFeatureSelectionTransformer), typeof(CountFeatureSelectionTransformer.Arguments), typeof(SignatureDataTransform),
+    CountFeatureSelectionTransformer.UserName, "CountFeatureSelectionTransform", "CountFeatureSelection")]
 
 namespace Microsoft.ML.Transforms
 {
     /// <include file='doc.xml' path='doc/members/member[@name="CountFeatureSelection"]' />
-    public static class CountFeatureSelectionTransform
+    public static class CountFeatureSelectionTransformer
```

Comment:
>CountFeatureSelectionTransformer [](start = 24, length = 32)

I see you've been renaming other files, which seems positive. Is there a reason to not rename these as well?